tests/ui/macros/issue-118048.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0121]: the placeholder `_` is not allowed within types on item signatures for functions
+  --> $DIR/issue-118048.rs:7:6
+   |
+LL | foo!(_);
+   |      ^
+   |      |
+   |      not allowed in type signatures
+   |      not allowed in type signatures
```

Comment:
It's worth noting that for this case, there's a duplicate label generated by ﻿`placeholder_types`. As we did previously, the method to eliminate the duplication is to add `﻿placeholder_types.dedup()`.